tests/index.test.ts

```diff
@@ -1,14 +1,14 @@
-import { test } from 'tap'
+import { test, TestContext } from 'node:test'
 import { join } from 'path'
 import { readFileSync } from 'fs'
 import Fastify from 'fastify'
 import fastifySession from '@fastify/secure-session'
 import querystring from 'querystring'
 import fastifyFlash from '../src'
 
-const key = readFileSync(join(__dirname, '..', 'secret-key'))
+const key = readFileSync(join(__dirname, '..', '..', 'secret-key'))
 
-test('should set error message and and clear up after displaying.', async t => {
+test('should set error message and and clear up after displaying.', async (t:TestContext) => {
```

Comment:
Why it requires explicit types?

Reply:
As we are using `t.assert` for assertions, it expects explicit type annotation.

Reply:
interesting, why would Node not know that the callback function to `test` is of type `TestContext`, do you know?

Reply:
It is actually the problem of TypeScript.
Using `assert` keyword will have some limitation that requires explicit `this`.

Reply:
Long story if you interested https://github.com/microsoft/TypeScript/issues/36931
Short Cap on https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70193#discussion_r1701867726

Reply:
Thanks @climba03003 . So we're happy with the explicit typing then?

Reply:
Yes